.github/workflows/push-tags.yaml

```
@@ -0,0 +1,28 @@
+name: Push Tag
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release version (e.g. v0.1.0)'
+        required: true
+
+defaults:
+  run:
+    # reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell
+    shell: bash --noprofile --norc -eo pipefail -x {0}
+
+jobs:
+  Create-Tag:
+    runs-on: ubuntu-18.04
+    if: {{ github.repository == 'rook/rook' && github.actor == 'travisn' }}
```

Do we need the actor? Any maintainer (with push access) should be able to run it, or at least can we check for the actor in a member of the maintainers group?
```suggestion
    if: {{ github.repository == 'rook/rook' && github.actor == 'travisn' }}
```

---

I think we can remove and option to trigger will be visible to the maintainer with push permission only.

>It's your fork so I don't see the button.

this comment from @leseb  also adds. we can will merge this and can confirm but logically also if someone don't have permission it should not show to them.